pkg/loader/load.go

```diff
@@ -142,6 +142,7 @@ func NewLoader(db *gosql.DB, opt ...Option) (Loader, error) {
 	}
 
 	db.SetMaxOpenConns(opts.workerCount)
+	db.SetMaxIdleConns(opts.workerCount)
```

Comment:
How about setting it to a lower value so that we don't need to keep many idle connections when drainer's not busy?

Reply:
should not do this, or you will need to create new connection every time the current use of `sql.DB` is bigger than the value you set.
but https://godoc.org/database/sql#DB.SetConnMaxLifetime can get what you want.

Reply:
I see. Your assumption is that `workerCount` will be fully used most of the time.
What I'm thinking about is if that is not the case, we would be wasting memory of drainer and the downstream database just to store connections that are going to timeout anyway.

Reply:
how's the cost of memory of for one conn? and how's the cost you create new connection every time?

Reply:
I mean it depends on the use ratio of `workerCount`.